src/components/topiccard.tsx

```diff
@@ -0,0 +1,34 @@
+import { forwardRef } from "react";
+import { cn } from "~/lib/utils";
+
+export interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
+  title: string;
+  variant?: "regular" | "glow";
+}
+
+const Card = forwardRef<HTMLDivElement, CardProps>(
+  ({ title, variant = "regular", className, ...other }, ref) => {
+    const variantClasses = {
+      regular: "bg-[#1D1D1D]",
+      glow: "bg-[#1D1D1D] [box-shadow:_0_0_15px_#50505088]",
```

Comment:
amount of glow is barely noticable, increase it slightly

Reply:
see the thing is, out background is black, and in a grey background it is not visible, but i suspect ti mmight be better in a dark background